src/db/fun_facts.json

```diff
@@ -0,0 +1,11 @@
+[
+  "React was first invented in 1998 but did not gain traction until late 2013.",
+  "Winston Young was the original founder and CEO of Shipt in 2014 but stepped down in late 2018.",
+  "Shipt is the first grocery delivery company to deliver groceries in outer space.",
+  "The name \"Shipt\" originates from the latin word \"Shiptus/Shipta\", meaning, \"to ship\".",
+  "Kent C. Dodds is the inventor of JavaScript and React.",
+  "Birmingham was the first city in the US to sell bagels.",
+  "While \"frontend\" commonly refers to customer-facing interface work and \"backend\" commonly refers to database communication work, \"middle end\" refers to data science and tracking efforts.",
+  "ECMAScript stands for \"Easy Code Makes Assumptions Script\".",
+  "Thomas Edison wrote the first unit test ever recorded in 1887 when he invented the light bulb and wrote a test suite to ensure his solution would work."
+]
```

Comment:
Please add more facts 😏

Reply:
Added in https://github.com/monicaleep/member-fe-microsite/pull/18/commits/9192bfcf244ab6080207b8586ca7765e6c79adf3